### ISSUE TITLE
Help using `focusOn`

### ISSUE THREAD

**Issue by pkaeding** [vasturiano/hilbert-chart#6]
First of all, thank you for publishing this. It looks like exactly what I am looking for, to be able to visualize our private IPv4 allocations.

However, I am having trouble using the `focusOn` function. All of our allocations are withing the 10/8 reserved space, so I would like to start zoomed in on that. Is that what `focusOn` is for? I might be misunderstanding the docs.

Here is an example I created to illustrate what I am seeing: https://observablehq.com/d/5995dc487d6ff315

I have a few netowrks defined in the CSV, and I am calling `focusOn`, but the chart is not zooming in, and the axis labels are gone. What am I doing wrong here?

**Comment by vasturiano** [comment]
@pkaeding sorry for the long wait on this, I finally had a chance to have a closer look.

Your assumption and approach is correct. The only thing that's missing is that you can only invoke `focusOn` **after** the chart has been rendered, which happens asynchronously. So, all you have to do is place inside a `setTimeout` right after you've initialized your chart. Something like:
```js
const myChart = HilbertChart() // ...

setTimeout(() => myChart.focusOn(privateSpace.firstIp().toNum(), privateSpace.countIps()));
```

Similar to how it's done in AS numbers example, to zoom in to the 16-bit portion only:
https://observablehq.com/@vasturiano/hilbert-map-of-autonomous-system-numbers-asn-allocations

**Comment by junousi** [comment]
Many thanks for the example @pkaeding and the timeout pointer @vasturiano. Just needed a tiny offset `focusOn(... , privateSpace.countIps()-1)` to get X/Y axis labels correct directly on page load, otherwise worked like a charm.

**Comment by junousi** [comment]
I tried to add [multiple charts](https://ipv4map.a3s.fi/rfc1918_test.html), but it seems when zooming in on them, the contents are swapped to that of the lastly defined chart.

@vasturiano Should it be technically possible to 

- have multiple charts on the same page
- have custom focusOn set per chart
- have zoom (in) operate in the focusOn specific context for each chart?

**Comment by vasturiano** [comment]
Thanks for your report @junousi. This has now been fixed in the latest version: `1.8.3`. 👍
